src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -21,7 +21,9 @@
           <ng-container *ngTemplateOutlet="logsLink"></ng-container>
           <ul>
             <li *ngFor="let check of healthData.health.checks">
-              <span [ngStyle]="check.severity | healthColor">{{ check.type }}</span>: {{ check.summary.message }}
+              <span [ngStyle]="check.severity | healthColor"
+                    [ngClass]="{'health-warn-description':check.severity === 'HEALTH_WARN'}">
```

Comment:
Oof, is this some type of angular behaviour? Idk how to read this xd.

Reply:
Hi @pereman2 sorry about that. It can be read as : 
if severity is health warning then add the class 'health-warn-descrition' to this span

Here is a much better description on the [angular docs](https://angular.io/api/common/NgClass)

Reply:
Hi @pereman2 please, is there another way this could have been done to make it simpler to understand?

Reply:
@pereman2 any suggestions? ^ Was this more a comment about the style of the code? Or is the way it is written acceptable?

Reply:
I think he just got mindblown by @nSedrickm's Angular skills 😜 Anyway, perhaps with a tiny whitespace addition it's easy to read that line: 
```suggestion
                    [ngClass]="{'health-warn-description': check.severity === 'HEALTH_WARN'}">
```
BTW, while the [`ngClass` helper](https://angular.io/api/common/NgClass) is usually the best way of doing this, there's also [another (perhaps clearer) way](https://stackoverflow.com/questions/45321526/difference-between-ngclass-vs-class-binding):
```suggestion
                    [class.health-warn-description]="check.severity === 'HEALTH_WARN'">
```

Reply:
Thanks @epuertat,  @pereman2 got dips on choice  here :)